woocommerce/woocommerce-gutenberg-products-block
# Active Filters: Fix active filter grouping for Ratings and Stock Status

This PR fixes the active filters grouping for Ratings and Stock Status under one heading.

Fixes #7574 

### Screenshots

| Before | After |
| ------ | ----- |
|![All_Products_–_woocommerce-blocks-2](https://user-images.githubusercontent.com/905781/199935263-edfb057f-16c6-4bdc-8602-9ae8e76df8d4.jpg)|![All_Products_–_woocommerce-blocks](https://user-images.githubusercontent.com/905781/199935320-45dbef23-ca11-4116-af1b-f7c7fe8b12d8.jpg)|

### Testing
1. Create a new page, add the all filters pattern and the All Products block.
2. Add some products that contain ratings and various stock statuses.
3. Filter by rating and stock status.
4. Confirm that the active filters are grouped under one heading.

#### User Facing Testing
The same as above.


### WooCommerce Visibility


### Changelog

> Active Product Filters: Fix active filter grouping for Product Ratings and Stock Statuses.
